data/defaults.yaml

```diff
@@ -328,3 +328,10 @@ prometheus::apache_exporter::package_ensure: 'latest'
 prometheus::apache_exporter::package_name: 'apache_exporter'
 prometheus::apache_exporter::user: 'apache-exporter'
 prometheus::apache_exporter::version: '0.7.0'
+prometheus::bind_exporter::download_url: 'https://github.com/digitalocean/bind_exporter/releases/download/v0.2.0-dev/bind_exporter'
```

Comment:
I think the idea is `download_url` is a way of a user overriding the url buit from the `download_url_base` and `version` parameters.  As such, it should default to `undef`.

Reply:
i don't know how this stuff works. i use debian packages. :p feel free to take over this PR as well, i couldn't figure out how to make the test suite work properly...